core/platform/models.py

```diff
@@ -27,8 +27,12 @@
 
 
 class _Platform(object):
+    """Parent Class for other classes to inherit
+    """
     @classmethod
     def import_models(cls):
+        """Function to defined in child classes.
```

Comment:
Change to-- Function to be implemented in child classes.

Reply:
Done